MD_bootcamp/1-Hoffman2_Cluster/3-How-to-submit-job.md

```diff
@@ -0,0 +1,53 @@
+# How to submit a job to Hoffman2 Cluster 
+Finally, you are going to submit your first simulation!!!
+
+There are diffent ways you can submit your job to Hoffman2 Cluster, let's first try the very basic one, and I will show you a better way to do it for next exercise.
+
+## Download input files and upload to Hoffman2
+Please go to ~/MD_bootcamp/5-Practice/1-SiO2 in this GitHub repo, and download all 4 files.  
```

Comment:
good to add a link to this folder

Reply:
How to do this before I merge?
Will the directory change after merging?
Can you help me edit this one?

Reply:
you may use a relative path, i.e. ../5-Practice/1-SiO2